demos/chat-client/Enarx.toml

```diff
@@ -13,3 +17,10 @@ kind = "connect"
 port = 50000
 prot = "tcp"
 host = "127.0.0.1"
+
+[[files]]
+name = "web"
+kind = "listen"
+port = 8080
```

Comment:
Let's use something more unique.
`50000` is already defined as the peer endpoint
How about `50010` for the web interface?